### PR TITLE
Locale review

### PR DESCRIPTION
**Pull request by rafamanzo**
Removed unused locales
Added double quotes to locales (except the autogenerated ones)
Fixed some wrong paths for locales

All the broken tests by these modifications were fixed in the subsequent commits.

### CODE REVIEW

**Comment by danielkza** [comment]
Almost everything looks good, except for some comments I left inline, and that we still have not cleaned up all the different ways of referring to model names (and the `new.female` vs. `new.male` thing).

File: app/views/kalibro_configurations/_kalibro_configuration.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <td><%= link_to t('show'), kalibro_configuration_path(kalibro_configuration.id), class: 'btn btn-info' %></td>
   <td>
     <% if kalibro_configuration_owner?(kalibro_configuration.id) %>
-      <%= link_to t('edit', model: ''), edit_kalibro_configuration_path(kalibro_configuration.id), class: 'btn btn-info' %>
+      <%= link_to t('edit'), edit_kalibro_configuration_path(kalibro_configuration.id), class: 'btn btn-info' %>
```
Comment (rafamanzo) [inline app/views/kalibro_configurations/_kalibro_configuration.html.erb]: Shouldn't we be using `t_action` here?

File: app/views/reading_groups/_reading.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <td style="background-color: #<%= reading.color %>; width: 20%;">&nbsp;</td>
   <% if reading_groups_owner? @reading_group.id %>
     <td>
-      <%= link_to t('edit', model: ''), edit_reading_group_reading_path(@reading_group.id, reading.id), class: 'btn btn-info' %>
+      <%= link_to t('edit'), edit_reading_group_reading_path(@reading_group.id, reading.id), class: 'btn btn-info' %>
```
Comment (rafamanzo) [inline app/views/reading_groups/_reading.html.erb]: Shouldn't we be using `t_action` here?

File: app/views/shared/_reading_groups_list.erb [hunk 3]
```diff
@@ -15,7 +15,7 @@
         <td><%= link_to t('show'), reading_group_path(reading_group.id), class: 'btn btn-info' %></td>
         <td>
           <% if reading_groups_owner? reading_group.id %>
-            <%= link_to t('edit', model: ''), edit_reading_group_path(reading_group.id), class: 'btn btn-info' %>
+            <%= link_to t('edit'), edit_reading_group_path(reading_group.id), class: 'btn btn-info' %>
```
Comment (rafamanzo) [inline app/views/shared/_reading_groups_list.erb]: Shouldn't we be using `t_action` here?

**Comment by rafamanzo** [comment]
Dear all, @pedroscocco is the only one not listed in this PR participants. So I suggest that we let him do the final review and accept the PR. What do you think?

**Comment by pedroscocco** [comment]
I just reviewed the PR with @marcheing and did some changes, but I could accept the PR as well.

**Comment by mezurometrics** [comment]
Good :)

Em ter, 5 de mai de 2015 às 11:23, pedroscocco <notifications@github.com>
escreveu:

File: app/views/shared/_reading_groups_list.erb (hunk 3 above)
Comment (pedroscocco) [inline app/views/shared/_reading_groups_list.erb]: We have decided that showing the model name in every row would be too repetitive

**Pull request merged**